src/main/resources/templates/home.html

```diff
@@ -17,6 +18,7 @@ <h2>Make new user</h2>
     dateOfBirth <input type="dateOfBirth" name="dateOfBirth">
```

Comment:
Change the type on this input to: `type="date"`.  This will allow the browser to automatically bring up a calendar for the user to select a date from, and it will consistently format it. For now, you can just store what you get from that input as a String instead of an int.